blocks/v2-forms/v2-forms.js

```diff
@@ -33,13 +33,18 @@ async function prepareRequest(form) {
   const url = form.dataset.action;
 
   const serializedData = serialize(payload);
-  loadScript(`${url}?${serializedData}`, { type: 'text/javascript', charset: 'UTF-8' });
+
+  return loadScript(`${url}&${serializedData}`, { type: 'text/javascript', charset: 'UTF-8' });
```

Comment:
Please update `&` with `?` and test submission again.

Reply:
Yup, it is working :) 
I've changed that before because there were 2 `?` characters in the URL...